.travis.yml

```diff
@@ -21,7 +21,7 @@ services:
 
 cache:
   directories:
-    - vendor
+    # - vendor
```

Comment:
I've put it back. I forgot it as I was trying to figure out why one of the tests was failing.